source/spark_sql_migrations/spark_sql_migrations/infrastructure/apply_migration_scripts.py

```diff
@@ -20,15 +20,18 @@ def _apply_migration_scripts(
 ) -> None:
     for migration in uncommitted_migrations:
         try:
-            table_versions = _get_table_versions()
+            if config.rollback_on_failure is True:
```

Comment:
Can it be simplified?
```suggestion
            if config.rollback_on_failure:
```